xdebug_handler_dbgp.c

```diff
@@ -2170,6 +2300,7 @@ static int xdebug_dbgp_parse_cmd(char *line, char **cmd, xdebug_dbgp_arg **ret_a
 	return XDEBUG_ERROR_DUP_ARG;
 }
 
+// -1 -> error, 0->normal, 1-> continue, 2->stop
```

Comment:
Please use constants for these, and not "magic numbers". This also would mean changing them in the additions above.